src/hotspot/share/opto/escape.cpp

```diff
@@ -2880,6 +2879,43 @@ bool ConnectionGraph::not_global_escape(Node *n) {
   return true;
 }
 
+/*
+ * The lock/unlock is unnecessary if we are locking a non-escaped object,
+ * unless synchronized block (defined by BoxLock node) has other escaped objects
+ * (for example, locked object come from Interpreter in OSR compilation).
+ *
+ * Return true if lock/unlock can be eliminated.
+ */
```

Comment:
```suggestion
// The lock/unlock is unnecessary if we are locking a non-escaped object,
// unless synchronized block (defined by BoxLock node) has other escaped objects
// (for example, locked object come from Interpreter in OSR compilation).
//
// Return true if lock/unlock can be eliminated.
```
This would be the first use in this file of multi-line comment :man_shrugging:

Reply:
Okay.